viz_v2/src/components/canvas/comments/CommentDetails.vue

```diff
@@ -74,9 +74,9 @@ export default {
     commentData (newVal, oldVal) {
       if (newVal !== null) {
         const d = newVal.comment
-
         this.pageType = d.page_title.startsWith('User') ? 'User page' : 'Talk page'
         this.pageTitle = d.page_title.split(':')[1]
+        this.pageLink = `https://en.wikipedia.org/wiki/${d.page_title.split(' ').join('_')}`
```

Comment:
hard coding 'en' is going to come back to haunt us.

Reply:
We are only using en.wikipedia conv data, right? Should I leave it out?

Reply:
This string should be injected from higher up - either as a language parameter or as the full http://en.wikipedia.org/wiki/...
My point was that you should no hard code strings deep in functions.

Reply:
Ah ok, changed!